.github/workflows/automatic-docs-update.yml

```diff
@@ -0,0 +1,63 @@
+name: Automatic docs update pipeline
+on:
+  issue_comment:
+    types: [created, edited]
+  # Run when PR is labeled
+  pull_request:
+    types: [labeled]
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: write
+  pages: write
+  id-token: write
+  pull-requests: write
+
+# Allow one concurrent deployment
+concurrency:
+  group: "pages"
+  cancel-in-progress: true
+
+jobs:
+  build-and-publish:
+    name: Automatically update docs
+    if: ${{ (github.event.issue.pull_request && contains(github.event.comment.body, '/refresh-docs')) || github.event.label.name == 'refresh-docs' }}
```

Comment:
@JKBGIT1

Reply:
Hi @cloudziu , you are correct about the condition, but not about the trigger. Adding the `refresh-docs` label to anything besides GH PR won't trigger the pipeline because of the `pull_request` part in the following code.
```
on:
  issue_comment:
    types: [created, edited]
  # Run when PR is labeled
  pull_request:
    types: [labeled]
```